host_core/native/hostcore_wasmcloud_native/src/lib.rs

```diff
@@ -104,6 +118,29 @@ rustler::init!(
     load = load
 );
 
+/// Create and store the NATS connection to be used for chunking
+/// invocations
+#[rustler::nif]
+fn set_chunking_connection_config(config: HashMap<String, String>) -> Result<Atom, Error> {
+    let nc = get_nats_connection(&config)?;
+    // TODO: add JS domain
+    let jsoptions = if let Some(d) = config.get("js_domain").cloned() {
+        JetStreamOptions::new().domain(&d)
+    } else {
+        JetStreamOptions::new()
+    };
+    let js = nats::jetstream::JetStream::new(nc, jsoptions);
+    let lattice = config
+        .get("lattice")
+        .cloned()
+        .unwrap_or("default".to_string());
```

Comment:
Does falling back actually help us recover here? Would it be better to return an error if there's no lattice prefix?

Reply:
Nearly everything in the wasmCloud ecosystem uses `default` as the lattice when one isn't supplied, so it felt natural to make this behave that way as well.